### PR TITLE
test: skip flaky logs tests

### PR DESCRIPTION
**Pull request by crenshaw-dev**
These constantly flake. I'd like to replace them with unit tests, just don't have the time right now.

image

### CODE REVIEW

File: test/e2e/app_management_test.go [hunk 1]
```diff
@@ -2058,32 +2058,6 @@ definitions:
 		})
 }
 
-func TestAppLogs(t *testing.T) {
```
Comment (blakepettersson) [inline test/e2e/app_management_test.go:2061]: Could it be worth adding a comment or todo to say that these tests should be re-implemented as unit tests?

Comment (ishitasequeira) [reply test/e2e/app_management_test.go:2061]: If we have plans to add tests for this, we should maybe create an issue to track it and add a TODO as well.. Else, we would forget it completely.

**Review by ishitasequeira — APPROVED** [review]
Thanks @crenshaw-dev for taking an action on this flaky tests!! Just a comment on creating a ticket to track and adding a TODO!!

File: test/e2e/app_management_test.go (hunk 1 above)
Comment (crenshaw-dev) [reply test/e2e/app_management_test.go:2061]: Done! https://github.com/argoproj/argo-cd/issues/13834

**Review by morey-tech — APPROVED** [review]

**Review by terrytangyuan — COMMENTED** [review]
Instead of removing, should we consider adding a link to an issue and skip on CI instead?

**Comment by crenshaw-dev** [comment]
@terrytangyuan I didn't see an existing "skip on CI" mechanism, so I just set to skip entirely. Folks generally don't run these locally anyway.

**Review by terrytangyuan — APPROVED** [review]
I meant something like checking env  var like this https://github.com/argoproj/argo-workflows/commit/9a2bb5e80d9f4e2d8792f31b57efbadc7ad41ef1

But this LGTM

**Review by blakepettersson — APPROVED** [review]

**Review by christianh814 — APPROVED** [review]

**Pull request merged**